test/MasterViewController.m

```diff
@@ -24,13 +28,19 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
     
-    self.org = [[Organisation alloc] initWithName:@"Corporation"];
-    [self.org addEmployeeWithName:@"Rork Smith"];
-    [self.org addEmployeeWithName:@"Rork2 Smith2"];
-    [self.org addEmployeeWithName:@"Rork3 Smith3"];
-    [self.org addEmployeeWithName:@"Rork4 Smith4"];
-    
-    self.title = @"employees";
+    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:@"Organisation"];
+
+    NSArray *fetchResult = [[DatabaseController sharedInstance].context executeFetchRequest:fetchRequest error:nil];
+    if (fetchResult.count > 0)
+    {   
+        self.org = [DatabaseController requestResultsForPredicate:(NSPredicate *)nil sortDescriptors:(NSArray *)nil entity:(NSString *)@"Organisation"].firstObject;
```

Comment:
1. You already have an array with organization, don't fetch it one more time.
2. You don't need to cast nil.